src/test/java/seedu/address/model/student/StudentTest.java

```diff
@@ -75,5 +81,10 @@ public void equals() {
         // different address -> returns false
         editedAlice = new StudentBuilder(ALICE).withAddress(VALID_ADDRESS_BOB).build();
         assertFalse(ALICE.equals(editedAlice));
+
+        editedAlice = new StudentBuilder(BOB)
```

Comment:
Hi, just checking if this is something extra that you did that isn't really related to the UI. Can I check if the purpose of this code is to check if the equals method works as expected?

Reply:
Yes, I wanted to check if the same student with different list would assert to be equals, however I had forgotten to change `BOB` back to `Alice` nice spot!